feat(content): Gegno Intro & Corroboration feedback & other adjustments (#11748)
endless-sky/endless-sky: data/gegno/gegno intro missions.txt

			branch "has met gegno"
				has "Visit Quarg in Gegno Space: offered"

			"After jumping through several systems of many alien ships, a Quarg ringworld is a surprisingly familiar sight. From the looks of it, this is a fully complete ringworld, a true sight to behold. Trillions of Quarg probably inhabit this ring, but what's more interesting is that other aliens are also present. They too are humanoid, and the most common figures you come across are either slightly above or below an average human's height. Their skin is a smooth, light gray, and adorned with different sorts of leather that remind you of medieval human clothing."
			`	The Quarg here move in an unusually quiet and eerie manner. They are dismissive of the other species walking about, who are equally as dismissive of you. The strange atmosphere is eventually broken when a few Quarg start to advance towards you, moving much quicker than the rest. After some unidentifiable discussion and gestures between themselves, one of them motions you to follow them.`
				goto walking
			
			label "has met gegno"
			action
				set "gegno: met gegno before quarg"

> 			`The visual of a complete Quarg Ringworld is drastically different than the alien homeworld you were on before. Such a pristine, advanced structure is quite the opposite of a disorderly industrial civilization. The population of Quarg here must be at least around the trillions, many times more than the population of the alien homeworld. What's even more interesting is there aliens of the kind you met before are here on the ring too. The Quarg must have some kind of relations with them, perhaps through trade, maintenance, or other services. Despite their presence, both they and the Quarg seem to be dismissive of one another, however, a few of the aliens occasionally glance over at you in some form of curiosity.`

Reviewer: ```suggestion
			`The visual of a complete Quarg Ringworld is drastically different than the alien homeworld you were on before. Such a pristine, advanced structure is quite the opposite of a disorderly industrial civilization. The population of Quarg here must be in the trillions, many times more than the population of the alien homeworld. What's even more interesting is that there are aliens of the kind you met before are here on the ring too. The Quarg must have some kind of relations with them, perhaps through trade, maintenance, or other services. Despite their presence, both they and the Quarg seem to be dismissive of one another. However, a few of the aliens occasionally glance over at you in some form of curiosity.`
```
Alternatively
```suggestion
			`The visual of a complete Quarg Ringworld is drastically different than the alien homeworld you were on before. Such a pristine, advanced structure is quite the opposite of a disorderly industrial civilization. The population of Quarg here must be in the trillions, many times more than the population of the alien homeworld. What's even more interesting is the aliens of the kind you met before are here on the ring too. The Quarg must have some kind of relations with them, perhaps through trade, maintenance, or other services. Despite their presence, both they and the Quarg seem to be dismissive of one another. However, a few of the aliens occasionally glance over at you in some form of curiosity.`
```

				decline

mission "Nosy Scin"
	invisible
	landing
	source "Giaru Gegno"
	to offer
		has "Visit Quarg in Gegno Space: offered"
		not "First Contact: Gegno Scin: offered"
		not "Gegno Genocide Defense: offered"
	to complete
		never
	on enter "Aleit"
		fail

	npc
		government "Gegno Scin (Neutral)"
		personality launching mute uninterested
		system "Aleit"
		ship "Mica" "Eichpys"

mission "Suspicious Scin"
	name `Strange Device`
	entering
	description `A strange Gegno device suddenly appeared on your ship, displaying coordinates to a planet within Gegno space. Visiting the planet might lead to some answers, or perhaps something else.`
	destination "Yiia Iyr"
	to offer
		"days since start" - "gegno: left quarg" > 5
		or
			and
				has "gegno: met gegno before quarg"
				has "Giaru Gegno: Quarg Contact: offered"
			has "Giaru Gegno: Revisit: offered"
		not "First Contact: Gegno Scin: offered"
		not "Gegno Genocide Defense: offered"
	on offer
		conversation

> 			`While you are going about your business, a faint signal starts emanating from somewhere in your ship. You aren't sure what it is, as it doesn't appear to originate from the ship itself, so you decide to take a look. After a few minutes of searching for the culprit, you come across a small device not too dissimilar from the one that was given to you by the Gegno you met on Var Oti, arbitrarily shoved into a corner of your ship's infrastructure. You don't know how or when it got there, but it doesn't appear to be dangerous like an explosive.`

Reviewer: ```suggestion
			`While you are going about your business, a faint signal starts emanating from somewhere in your ship. You aren't sure what it is, as it doesn't appear to originate from the ship itself, so you decide to take a look. After a few minutes of searching for the culprit, you come across a small device not too dissimilar from the one that was given to you by the Gegno you met on Var Oti, arbitrarily shoved into a corner of your ship's infrastructure. You don't know how or when it got there, but it doesn't appear to be dangerous.`
```

		not "Gegno Genocide Defense: offered"
	on offer
		conversation

> 			`Lead here by the mysterious device on your ship, you descent to the surface of this world, going through several thick layers of clouds. Once you break though them and prepare to land, you notice what appear to be a few Gegno Scin ships grounded nearby, easily distinguished by their various green color patterns. Two of them look like military vessels you've seen in nearby systems, while a third one between them is a small shuttlecraft. The Scin must be responsible for the device, seemingly expecting you as one of their warships begins to lift off the ground and head in your direction.`

Reviewer: ```suggestion
			`Led here by the mysterious device on your ship, you descend to the surface of this world, going through several thick layers of clouds. Once you break though them and prepare to land, you notice what appear to be a few Gegno Scin ships grounded nearby, easily distinguished by their various green color patterns. Two of them look like military vessels you've seen in nearby systems, while a third one between them is a small shuttlecraft. The Scin must be responsible for the device, seemingly expecting you as one of their warships begins to lift off the ground and head in your direction.`
```

			label "leave"
			action
				set "gegno: left scin"

> 			`	Wary of the Scin warship, you turn around and begin to ascend back into space. The Scin ship advances towards yours, but as it comes closer, it begins to take on an escort formation with you, following you as you leave the planet's atmosphere. You're unsure of what they're doing on the planet or why you've been lead to them, but maybe it's better for you to be careful and not get involved.`

Reviewer: ```suggestion
			`	Wary of the Scin warship, you turn around and begin to ascend back into space. The Scin ship advances towards yours, but as it comes closer, it begins to take on an escort formation with you, following you as you leave the planet's atmosphere. You're unsure of what they're doing on the planet or why you've been led to them, but maybe it's better for you to be careful and not get involved.`
```

			`	Still, it might be best to listen to what the Quarg said and exercise caution going forward.`
				accept
			
			label provocative
			`The Scin ships following you off the planet show little hesitation before firing their weapons at you, likely feeling provoked by your choice of actions with them during the encounter. Still, they must have some sort of interest in you if they led you to them, meaning that there could be other possibilities to get involved with the Gegno at other locations in their space.`
			`	Still, the Gegno already have qualms with outsiders. It might be a good idea to listen to what the Quarg said about exercising caution, especially if it leads to hostile encounters.`
				accept

# Fails above mission once the proper conditions are met so that it doesn't float in the mission panel
mission "Gegno Exploration Nudge Remover"
	invisible

> 	landing

Reviewer 2: ```suggestion
	invisible
	landing
	non-blocking
```

		has "Gegno Exploration Nudge Remover: done"
	on enter
		conversation
			branch provocative
				has "scin: provocative"
			`As you leave the planet's atmosphere, the Scin ship breaks formation with you and goes their separate way. While the Gegno may not be interested in associating with you overall, it seemed like the few Scin you met were interested in some capacity. There may be other opportunities to encounter the Gegno elsewhere in their space and learn more about them.`
			`	Still, it might be best to listen to what the Quarg said and exercise caution going forward.`
				accept
			
			label provocative
			`The Scin ships following you off the planet show little hesitation before firing their weapons at you, likely feeling provoked by your choice of actions with them during the encounter. Still, they must have some sort of interest in you if they led you to them, meaning that there could be other possibilities to get involved with the Gegno at other locations in their space.`
			`	Still, the Gegno already have qualms with outsiders. It might be a good idea to listen to what the Quarg said about exercising caution, especially if it leads to hostile encounters.`
				accept

> # Fails above mission once the proper conditions are met so that it doesn't float in the mission panel

Reviewer 2: ```suggestion
# Fails above mission once the proper conditions are met so that it doesn't float in the mission panel.
```
